packages/deployer/src/infrastructure/aws.ts

```diff
@@ -0,0 +1,46 @@
+import AWS from 'aws-sdk';
+
+async function deleteObjects(s3: AWS.S3, bucket: string, objects: AWS.S3.DeleteMarkerEntry[]) {
+  await s3
+    .deleteObjects({
+      Bucket: bucket,
+      Delete: {
+        Objects: objects.map(({ Key, VersionId }) => ({ Key, VersionId })),
+      },
+    })
+    .promise();
+}
+
+async function fetchObjects(s3: AWS.S3, bucket: string) {
+  const objects = await s3.listObjectVersions({ Bucket: bucket }).promise();
+  return [...objects.Versions, ...objects.DeleteMarkers];
+}
+
+async function emptyBucket(s3: AWS.S3, bucket: string) {
+  let objects = await fetchObjects(s3, bucket);
+
+  // Objects are fetch by max of 1000 in one batch
+  while (objects.length > 0) {
+    await deleteObjects(s3, bucket, objects);
+
+    objects = await fetchObjects(s3, bucket);
+  }
+}
+
+async function deleteBucket(s3: AWS.S3, bucket: string) {
```

Comment:
You mention in the PR description that you can't remove the bucket. I am confused now.